### PR TITLE
:heart: give love

### PR DESCRIPTION
**Pull request by rhcarvalho**
@mfojtik giving love to your script... ptal

@php-coder FYI :eyes:

### CODE REVIEW

File: hack/rebase-to-origin.sh [hunk 1]
```diff
@@ -7,10 +7,22 @@
 #       present in the pkg/ folder.
 #
 
-source "${s2i_dir}/hack/util.sh"
```
Comment (rhcarvalho) [inline hack/rebase-to-origin.sh]: This was broken :smile:

**Comment by rhcarvalho** [comment]
@bparees ptal here too. I'm gonna need to bump s2i in origin again, would be nice to have this in master :)

File: hack/rebase-to-origin.sh [hunk 2]
```diff
@@ -19,29 +31,27 @@ exclude_pkgs=(
...
-  git checkout -b "s2i-${s2i_short_ref}-bump"
+pushd "${OS_ROOT}" >/dev/null
+  git checkout -B "s2i-${s2i_short_ref}-bump" master
   rm -rf "${origin_s2i_godep_dir}/*"
-  cp -R ${s2i_dir}/pkg ${origin_s2i_godep_dir}/.
+  cp -R "${S2I_ROOT}/pkg" "${origin_s2i_godep_dir}/."
 
   # Remove all test files
-  find ${origin_s2i_godep_dir} -type f -name '*_test.go' -delete
+  find "${origin_s2i_godep_dir}" -type f -name '*_test.go' -delete
```
Comment (bparees) [inline hack/rebase-to-origin.sh]: let's hold on merging this until we find out if we should be removing the test files or not.

**Comment by rhcarvalho** [comment]
@bparees updated the changes to not remove test files, mirroring the behavior of `godep`.

**Comment by bparees** [comment]
lgtm [merge]

**Pull request merged**